data-collection/src/main/java/refactoringml/util/RefactoringUtils.java

```diff
@@ -362,4 +362,15 @@ public static boolean isStudied(Refactoring refactoring){
 		return refactorings.stream().filter(r -> renameTypes.contains(r.getRefactoringType())).collect(Collectors.toList());
 	}
 
+
+	/**
+     * We detect anonymous classes by means of a simple heuristic: if the last part of a full class
+     * name starts with a lower case, odds are that it's an anonymous class.
+	 */
+	public static boolean isAnonymousClass(String fullName) {
+		String[] parts = fullName.split("\\.");
```

Comment:
oh, this can throw an exception if, for some reason, we get something that doesn't comply with the format. 

I'll write a test for a class without package name.

Hard to guess, but I'd say other cases should not happen. But we'll only know when we run.